examples/https-nginx/make_secret.go

```diff
@@ -63,5 +63,5 @@ func main() {
 			"nginx.key": nginxKey,
 		},
 	}
-	fmt.Printf(runtime.EncodeOrDie(latest.GroupOrDie("").Codec, secret))
+	fmt.Printf(runtime.EncodeOrDie(testapi.Default.Codec(), secret))
```

Comment:
@bprashanth, latest doesn't work because we separate the logic that installs the API into other packages: `k8s.io/kubernetes/pkg/api/install` and `k8s.io/kubernetes/pkg/apis/experimental/install`. testapi imports these, but latest doesn't. I think instead of using testapi in the example, it's better just import the install packages.